app/_includes/sponsors.html

```diff
@@ -21,9 +21,9 @@
   </div>
   <div class="row">
     <div class="partners premium-partners">
-      <div class="col-md-offset-4 col-sm-6 col-md-4 partner">
-        <a href="http://www.amdtelecom.net/" target="_blank">
-          <img src="images/sponsors/amd-telecom_logo.jpg" alt="AMD Telecom" class="img-responsive">
+      <div class="col-sm-6col-md-12 partner">
```

Comment:
@modkaffes You also need to edit the Site's menu to include the Sponsors section.